README.md

```diff
@@ -122,6 +121,53 @@ to feed into a SIEM" use case. By default, the script sleeps for 300 seconds and
 If you change query parameters between runs, update the fields being requested, or make other
 modifications you will need to erase the `~/.tc3/tc-portal-end-cursor` file and start from scratch.
 
+## Event Filtering
+The events API allows filtering on many event, formula, and customer/connector related fields, and it supports
```

Comment:
is connector  a term our customers will know?  havent seen how we describe the connections between us and them.

Reply:
We don't/haven't and it's not a great term, but it's the best we could come up with. It's a pretty standard term in the SASE space to describe connections from customer to provider, so I'm hoping people figure out what it means.